### PR TITLE
fix(promise): reduce bottom space by switching to flex

### PR DESCRIPTION
**Pull request by muryoh**
https://backlight.dev/preview/1SuMaEt5uxgp62VC8WVP/promise?branch=promise-fix@yYX2ZXo57rSdHQUA7FQ2iG3lYGs1

### CODE REVIEW

**Comment by muryoh** [comment]
Preview: https://dev-designsystems--promise-fix-g4sa2kfe.web.app/

File: promise/src/promise.vue [hunk 1]
```diff
@@ -14,14 +14,14 @@
 </script>
 <template>
   <div data-aos="fade-up"
-    :class="`max-w-6xl grid ${layout === 'horizontal' ? 'mx-auto md:grid-cols-2 gap-12' : 'md:grid-rows-2 text-center'} place-items-center py-10`">
+    :class="`max-w-6xl flex flex-wrap ${layout === 'horizontal' ? 'flex-row mx-auto justify-center gap-12' : 'flex-col text-center'} place-items-center py-10`">
```
Comment (gqio) [inline promise/src/promise.vue:17]: we need both mx-auto and justify-center ? I thought the container will take all the width

Comment (gqio) [reply promise/src/promise.vue:17]: Ohhhhh you mean my-auto maybe for collapsin margin ?

Comment (muryoh) [reply promise/src/promise.vue:17]: div has **max-w-6xl**, so it has a set width: without **mx-auto** it stays on the left
Then inside the div I need elements to be centered with flex's justify-center

You tell me if I got it wrong, you're the CSS king :pray:

Comment (gqio) [reply promise/src/promise.vue:17]: I got it thanks for the heads up I missed the max-w for widescreen

**Review by gqio — APPROVED** [review]

**Pull request merged**